testdata/offBy1/positive_tests.go

```diff
@@ -0,0 +1,8 @@
+package checker_test
+
+func lenIndex(xs []int, ys []string) {
+	/*! index expr always panics; maybe you wanted xs[len(xs)-1]? */
+	_ = xs[len(xs)]
+	/*! index expr always panics; maybe you wanted ys[len(ys)-1]? */
+	_ = ys[len(ys)]
```

Comment:
What about an array example ?

Reply:
It's compile time error.

Reply:
hah, wasn't 100% sure about that and was lazy to check it. nice.